Assets/MRTK/Core/Interfaces/InputSystem/IMixedRealityInputSystem.cs

```diff
@@ -119,6 +120,16 @@ public interface IMixedRealityInputSystem : IMixedRealityEventSystem
         /// </summary>
         void ClearFallbackInputStack();
 
+        /// <summary>
+        /// Interface for subscribing handlers to a specific phase of the event propagation
```

Comment:
In a future PR, it would be helpful here to either explain the phases of event propagation supported, or to link to a documentation page which explains the phases supported.